roles/obol_charon/tasks/main.yml

```diff
@@ -27,6 +27,9 @@
         secret_destination_path: "{{ obol_charon_private_key_file_path }}"
         secret_type: key
 
+# Get cluster lock file 
+# And also the validator_keys files from vault ?
```

Comment:
yeah lets add another vault function, also before using a loop, check if there is any way to fecth the entire directory in one go